--- ui_settings_controller.org.sso	Sun May 24 08:34:26 2020
+++ ui_settings_controller.sso	Sun May 24 08:35:26 2020
@@ -1040,7 +1040,7 @@
       header   =   "UI_SETTINGS_FIRST_PERSON_CAMERA_FOV"
       desc   =   "UI_SETTINGS_FIRST_PERSON_CAMERA_FOV_DESC"
       minValue   =   80
-      maxValue   =   130
+      maxValue   =   180
       defaultValue   =   107
       stepInterval   =   1
       textfield   =   "INTEGER"
@@ -1051,7 +1051,7 @@
       header   =   "UI_SETTINGS_THIRD_PERSON_CAMERA_FOV"
       desc   =   "UI_SETTINGS_THIRD_PERSON_CAMERA_FOV_DESC"
       minValue   =   80
-      maxValue   =   130
+      maxValue   =   180
       defaultValue   =   80
       stepInterval   =   1
       textfield   =   "INTEGER"
